src/Api/Management/Entity/AppInterface.php

```diff
@@ -83,4 +83,12 @@ public function setCallbackUrl(string $callbackUrl): void;
      * @return \Apigee\Edge\Api\Management\Entity\AppCredentialInterface[]
      */
     public function getCredentials(): array;
+
+    /**
+     * Set Initial API Product while creating App.
+     *
+     * @param array $initialApiProducts
+     *
+     */
+    public function setInitialApiProducts(array $initialApiProducts): void;
```

Comment:
Generic advice: changing interfaces MUST NOT happen in minor/patch releases. 

Keeping this here with a `@todo` _could_ be a workaround...